src/object/object_representation.c

```diff
@@ -4491,7 +4491,12 @@ or_packed_domain_size (TP_DOMAIN * domain, int include_classoids)
 	case DB_TYPE_JSON:
 	  if (d->json_validator != NULL)
 	    {
-	      size += or_packed_string_length (db_json_get_schema_raw_from_validator (d->json_validator), NULL);
+	      DB_VALUE temp;
+	      // *INDENT-OFF*
+	      db_make_string (&temp, const_cast <char*>(db_json_get_schema_raw_from_validator (d->json_validator)));
```

Comment:
Please add or_packed_json_schema_length and or_packed_json_validator_length functions.